flight/PiOS/inc/pios_can.h

```diff
@@ -35,6 +35,7 @@
 
 //! The set of CAN messages
 enum pios_can_messages {
+	PIOS_CAN_ERROR = -1,
```

Comment:
>  Of course, this warning is wrong, since C does not prevent you from taking a variable foo and assigning any value you like to it, i.e. there's no GCC protection* that only enum values can be assigned to enum variables.

This is actually correct; from the C99 standard:

>  4 Each enumerated type shall be compatible with char, a signed integer type, or an unsigned integer type. The choice of type is implementation-defined,110) but shall be capable of representing the values of all the members of the enumeration.